web/concrete/authentication/concrete/controller.php

```diff
@@ -1,7 +1,8 @@
 <?php
-namespace Concrete\Authentication\Concrete;
+namespace concrete\authentication\concrete;
```

Comment:
@ignasigarcia we abide by PSR-4 which requires namespace segments match their directory name and prefix exactly. If we left this lowercase the file would never autoload on case sensitive systems.